app/logic/ui/reactableHelper.R

```diff
@@ -181,57 +210,75 @@ indexReactable <- function(currentData){
             class = "stickyReactableColumn",
             headerClass = "stickyReactableHeader",
             searchable = TRUE,
-            cell = function(value, index){
+            cell = function(value, index) {
               clubLogos(value, index, currentData)
             }
           ),
-          club = 
-            colDef(
-              show = FALSE,
-              searchable = TRUE
-            )
-        ) |> 
+          club =
+          colDef(
+            show = FALSE,
+            searchable = TRUE
+          )
+        ) |>
         append(
           pmap(statisticsTooltips, ~ {
-            if((..1) %in% names(currentData)) {
-              ..1 =
+            if ((..1) %in% names(currentData)) {
+              ..1 <- # nolint: object_name_linter
```

Comment:
Not sure why how the `..1` object name works so I didn't want to mess with it. The linter complained it isn't following camel case.